documentation/src/main/asciidoc/ch06.asciidoc

```diff
@@ -173,6 +173,19 @@ It is important to add each configured constraint violation by calling `addConst
 Only after that the new constraint violation will be created.
 ====
 
+[WARNING]
+====
+Note that the custom message template is passed directly to the Expression Language engine.
+
+Thus, you should be very careful when integrating user input in a custom message template as it will be interpreted
+by the Expression Language engine, which is usually not the behavior you expect and could cause a sensitive data leak.
```

Comment:
My first thought reading this was "what is a sensitive leak?". I would use "leak of sensitive data" to avoid the confusion, or "could cause sensitive data to leak", or even "could allow malicious users to leak sensitive data", but it's your call.

Reply:
Your second proposal is very nice. Changed, thanks!